docs/migrate-swagger/01-get-started.md

```diff
@@ -73,6 +73,18 @@ This is the probably most critical step of the conversion. As you have pre-sorte
 
 - Review CI checks such as breaking changes and other failures.
 
+## Troubleshooting
+
+### Error "does not have a GetAll method"
+
+If converter fails withe error
+
+```
+fatal   | The ResourceCollection RestorePointCollection (RequestPath: {Certain request path}) does not have a GetAll method
+```
+
+you could bypass it by the configuration in this [page](https://github.com/Azure/autorest.csharp/blob/feature/v3/docs/mgmt/readme.md#list-exception).
```

Comment:
No quite sure about the action here.
By checking the link, do you suggest to update the config or update the Swagger to solve the problem?